base/tests/test_OperationMultipleEvalInterModLinear.cpp

```diff
@@ -26,28 +27,28 @@ BOOST_AUTO_TEST_SUITE(TestOperationMultipleEvalInterModLinear)
 BOOST_AUTO_TEST_CASE(zeroAlpha) {
   srand(static_cast<unsigned>(time(nullptr)));
   const size_t dim = size_t(4+10.*rand()/static_cast<double>(RAND_MAX));
-  std::vector<std::vector<size_t>> interactions = std::vector<std::vector<size_t>>();
+  std::set<std::set<size_t>> interactions = std::set<std::set<size_t>>();
```

Comment:
It's not necessary to initialize an empty `set` and copy it over to `tmp`. Just do
```suggestion
  std::set<std::set<size_t>> interactions;
```
(uses default constructor), similarly for the others occurrences.

Reply:
There are still some left.